package.json

```diff
@@ -107,19 +107,20 @@
   },
   "keywords": [],
   "dependencies": {
-    "@angular/animations": "^8.0.0",
-    "@angular/cdk": "^8.0.0",
-    "@angular/common": "^8.0.0",
-    "@angular/compiler": "^8.0.0",
-    "@angular/core": "^8.0.0",
-    "@angular/forms": "^8.0.0",
-    "@angular/material": "^8.0.0",
-    "@angular/platform-browser": "^8.0.0",
-    "@angular/platform-browser-dynamic": "^8.0.0",
-    "@angular/router": "^8.0.0",
-    "@angular/upgrade": "^8.0.0",
-    "@ng-bootstrap/ng-bootstrap": "^5.3.1",
+    "@angular/animations": "^10.2.2",
+    "@angular/cdk": "^10.2.2",
+    "@angular/common": "^10.2.2",
+    "@angular/compiler": "^10.2.2",
+    "@angular/core": "^10.2.2",
+    "@angular/forms": "^10.2.2",
+    "@angular/material": "^10.2.2",
+    "@angular/platform-browser": "^10.2.2",
+    "@angular/platform-browser-dynamic": "^10.2.2",
+    "@angular/router": "^10.2.2",
+    "@angular/upgrade": "^10.2.2",
+    "@ng-bootstrap/ng-bootstrap": "^8.0.0",
     "@types/angular-translate": "^2.16.2",
+    "acorn": "6.4.2",
```

Comment:
Hi, any reason why is this included without `^`? Is there a possibility to upgrade this?

Reply:
Hey there's no reason, must have been a command-line typo. Feel free to change as necessary!